sonar-project.properties

```diff
@@ -1,11 +1,12 @@
 sonar.host.url=http://localhost:9000 
 sonar.login=admin
-sonar.password=admin
+sonar.password=pdx-admin-ui
```

Comment:
Can you link to information in the Confluence or in the readme how to run SonarQube locally for the project?

Reply:
This is the link in confluence: https://www.ebi.ac.uk/seqdb/confluence/display/MouseInformatics/PDX+Admin.
I think putting it in readme is also a good idea but I'll do that in another PR